docs/_static/vdr/v1.yaml

```diff
@@ -77,13 +77,39 @@ paths:
               $ref: '#/components/schemas/DIDUpdateRequest'
       responses:
         "200":
-          description: DID Document has been updated.
+          description: DID document has been updated.
           content:
             application/json+did-document:
               schema:
                  $ref: '#/components/schemas/DIDDocument'
         "400":
-          description: DID couldn't be updated because the input conflicts or is malformed.
+          description: DID document couldn't be updated because the did param was malformed or the document was deactivated.
```

Comment:
I agree with Rein that a 409 should be supported. A 409 indicates you should not repeat the request even with other parameters where a 400 suggests that you can repeat is with other parameters.

Reply:
Ok, I 200 the 409 👍